cmd/mmv/main.go

```diff
@@ -100,7 +100,11 @@ func rename(args []string) error {
 		return err
 	}
 	defer tty.Close()
-	cmd := exec.Command(editor, f.Name())
+
+	editorWithArgs := strings.Split(editor, " ")
```

Comment:
Please change this split to `strings.Fields`.

Reply:
👍  fixed fce8377

Reply:
I didn't knew `strings.Fields`. Thanks!